dev/com.ibm.ws.ssl/resources/com/ibm/ws/ssl/resources/ssl.nlsprops

```diff
@@ -656,4 +656,8 @@ ssl.client.handshake.cacerts.error.CWPKI0828E.useraction=Review the extended err
 ssl.update.certificate.error=CWPKI0829E: SSL certificate update error. {0}
 ssl.update.certificate.error.explanation=The SSL certificate could not be updated at the specified location.
 ssl.update.certificate.error.useraction=Ensure the location is accessible by the server process.  Review the FFDC logs for additional errors that are associated with generating or loading the keys.
+# -------------------------------------------------------------------------------------------------
+ssl.certificate.add.CWPKI0830I=CWPKI0830I: Certificate with SubjectDN {0} from the environment variable {1} is being added to the {2} keystore.
+ssl.certificate.add.CWPKI0830I.explanation=An informative message to indicate a certificate from an environment variable is being added to a keystore.
```

Comment:
does this message appear _before_ the cert is added, ie does it notify the user that the process is unbderway or that the cert is successfully added?

Reply:
The message issued just before I write the certificate to the keystore file.

Reply:
ok- for the explanation, can we give the user info about the flow of events, ie

`A certificate that was created by an environmental variable in the configuration is in the process of being added to the keystore file.`

Reply:
@dmuelle  I'll make the changes to the message but now that I think about it "A certificate that was created by an environmental variable in the configuration is in the process of being added to the keystore file." really is not correct for the explanation.  We are not creating the certificate we are actually getting the certificate from the environment variable.  Perhaps "A certificate that was retrieved from an environmental variable in the configuration is in the process of being added to the keystore file."  ?

Reply:
yes- actually now that I look at it, you can simplify it and just say:

A certificate was retrieved from an environmental variable in the configuration and is being added to the keystore file.